2024-01-22 09:12:00,071 INFO    StreamThr :3635583 [internal.py:wandb_internal():86] W&B internal server running at pid: 3635583, started at: 2024-01-22 09:12:00.071496
2024-01-22 09:12:00,073 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: status
2024-01-22 09:12:00,079 INFO    WriterThread:3635583 [datastore.py:open_for_write():85] open: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/run-ke2nwbty.wandb
2024-01-22 09:12:00,080 DEBUG   SenderThread:3635583 [sender.py:send():379] send: header
2024-01-22 09:12:00,080 DEBUG   SenderThread:3635583 [sender.py:send():379] send: run
2024-01-22 09:12:01,043 INFO    SenderThread:3635583 [dir_watcher.py:__init__():211] watching files in: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files
2024-01-22 09:12:01,043 INFO    SenderThread:3635583 [sender.py:_start_run_threads():1121] run started: ke2nwbty with start time 1705914720.070765
2024-01-22 09:12:01,044 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: summary_record
2024-01-22 09:12:01,044 INFO    SenderThread:3635583 [sender.py:_save_file():1375] saving file wandb-summary.json with policy end
2024-01-22 09:12:01,055 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: check_version
2024-01-22 09:12:01,056 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: check_version
2024-01-22 09:12:01,810 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: run_start
2024-01-22 09:12:01,838 DEBUG   HandlerThread:3635583 [system_info.py:__init__():31] System info init
2024-01-22 09:12:01,838 DEBUG   HandlerThread:3635583 [system_info.py:__init__():46] System info init done
2024-01-22 09:12:01,838 INFO    HandlerThread:3635583 [system_monitor.py:start():181] Starting system monitor
2024-01-22 09:12:01,838 INFO    SystemMonitor:3635583 [system_monitor.py:_start():145] Starting system asset monitoring threads
2024-01-22 09:12:01,838 INFO    HandlerThread:3635583 [system_monitor.py:probe():201] Collecting system info
2024-01-22 09:12:01,839 INFO    SystemMonitor:3635583 [interfaces.py:start():190] Started cpu monitoring
2024-01-22 09:12:01,840 INFO    SystemMonitor:3635583 [interfaces.py:start():190] Started disk monitoring
2024-01-22 09:12:01,842 INFO    SystemMonitor:3635583 [interfaces.py:start():190] Started gpu monitoring
2024-01-22 09:12:01,843 INFO    SystemMonitor:3635583 [interfaces.py:start():190] Started memory monitoring
2024-01-22 09:12:01,845 INFO    SystemMonitor:3635583 [interfaces.py:start():190] Started network monitoring
2024-01-22 09:12:01,920 DEBUG   HandlerThread:3635583 [system_info.py:probe():195] Probing system
2024-01-22 09:12:01,921 DEBUG   HandlerThread:3635583 [gitlib.py:_init_repo():53] git repository is invalid
2024-01-22 09:12:01,921 DEBUG   HandlerThread:3635583 [system_info.py:probe():240] Probing system done
2024-01-22 09:12:01,921 DEBUG   HandlerThread:3635583 [system_monitor.py:probe():210] {'os': 'Linux-5.15.0-83-generic-x86_64-with-glibc2.35', 'python': '3.10.12', 'heartbeatAt': '2024-01-22T09:12:01.920309', 'startedAt': '2024-01-22T09:12:00.063795', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '/workspace/UPRTH/sweep.py', 'codePath': 'sweep.py', 'host': '470e8c8e5128', 'username': 'root', 'executable': '/bin/python3', 'cpu_count': 32, 'cpu_count_logical': 64, 'cpu_freq': {'current': 2481.8361562499995, 'min': 800.0, 'max': 3500.0}, 'cpu_freq_per_core': [{'current': 800.0, 'min': 800.0, 'max': 3500.0}, {'current': 1100.0, 'min': 800.0, 'max': 3500.0}, {'current': 2962.764, 'min': 800.0, 'max': 3500.0}, {'current': 2896.031, 'min': 800.0, 'max': 3500.0}, {'current': 2895.93, 'min': 800.0, 'max': 3500.0}, {'current': 2898.547, 'min': 800.0, 'max': 3500.0}, {'current': 2883.632, 'min': 800.0, 'max': 3500.0}, {'current': 2876.167, 'min': 800.0, 'max': 3500.0}, {'current': 2973.971, 'min': 800.0, 'max': 3500.0}, {'current': 2894.341, 'min': 800.0, 'max': 3500.0}, {'current': 2547.281, 'min': 800.0, 'max': 3500.0}, {'current': 2905.453, 'min': 800.0, 'max': 3500.0}, {'current': 3051.846, 'min': 800.0, 'max': 3500.0}, {'current': 1300.0, 'min': 800.0, 'max': 3500.0}, {'current': 2920.819, 'min': 800.0, 'max': 3500.0}, {'current': 2924.301, 'min': 800.0, 'max': 3500.0}, {'current': 2920.295, 'min': 800.0, 'max': 3500.0}, {'current': 2790.512, 'min': 800.0, 'max': 3500.0}, {'current': 2925.366, 'min': 800.0, 'max': 3500.0}, {'current': 2894.168, 'min': 800.0, 'max': 3500.0}, {'current': 2927.61, 'min': 800.0, 'max': 3500.0}, {'current': 2872.579, 'min': 800.0, 'max': 3500.0}, {'current': 2946.464, 'min': 800.0, 'max': 3500.0}, {'current': 3500.0, 'min': 800.0, 'max': 3500.0}, {'current': 2858.054, 'min': 800.0, 'max': 3500.0}, {'current': 2902.464, 'min': 800.0, 'max': 3500.0}, {'current': 3500.0, 'min': 800.0, 'max': 3500.0}, {'current': 2777.97, 'min': 800.0, 'max': 3500.0}, {'current': 2893.182, 'min': 800.0, 'max': 3500.0}, {'current': 2909.129, 'min': 800.0, 'max': 3500.0}, {'current': 2893.149, 'min': 800.0, 'max': 3500.0}, {'current': 2903.046, 'min': 800.0, 'max': 3500.0}, {'current': 2689.734, 'min': 800.0, 'max': 3500.0}, {'current': 2800.0, 'min': 800.0, 'max': 3500.0}, {'current': 800.0, 'min': 800.0, 'max': 3500.0}, {'current': 2893.417, 'min': 800.0, 'max': 3500.0}, {'current': 2892.924, 'min': 800.0, 'max': 3500.0}, {'current': 2858.783, 'min': 800.0, 'max': 3500.0}, {'current': 900.0, 'min': 800.0, 'max': 3500.0}, {'current': 900.0, 'min': 800.0, 'max': 3500.0}, {'current': 2934.779, 'min': 800.0, 'max': 3500.0}, {'current': 2891.904, 'min': 800.0, 'max': 3500.0}, {'current': 1200.0, 'min': 800.0, 'max': 3500.0}, {'current': 2912.986, 'min': 800.0, 'max': 3500.0}, {'current': 800.0, 'min': 800.0, 'max': 3500.0}, {'current': 2797.593, 'min': 800.0, 'max': 3500.0}, {'current': 2977.856, 'min': 800.0, 'max': 3500.0}, {'current': 2917.929, 'min': 800.0, 'max': 3500.0}, {'current': 2935.558, 'min': 800.0, 'max': 3500.0}, {'current': 3500.0, 'min': 800.0, 'max': 3500.0}, {'current': 2945.13, 'min': 800.0, 'max': 3500.0}, {'current': 2890.366, 'min': 800.0, 'max': 3500.0}, {'current': 2982.076, 'min': 800.0, 'max': 3500.0}, {'current': 3500.0, 'min': 800.0, 'max': 3500.0}, {'current': 3500.0, 'min': 800.0, 'max': 3500.0}, {'current': 1913.692, 'min': 800.0, 'max': 3500.0}, {'current': 2400.0, 'min': 800.0, 'max': 3500.0}, {'current': 800.0, 'min': 800.0, 'max': 3500.0}, {'current': 2866.36, 'min': 800.0, 'max': 3500.0}, {'current': 800.0, 'min': 800.0, 'max': 3500.0}, {'current': 800.0, 'min': 800.0, 'max': 3500.0}, {'current': 2915.475, 'min': 800.0, 'max': 3500.0}, {'current': 2919.636, 'min': 800.0, 'max': 3500.0}, {'current': 1000.0, 'min': 800.0, 'max': 3500.0}], 'disk': {'total': 1758.8579597473145, 'used': 946.0875854492188}, 'gpu': 'NVIDIA A800-SXM4-80GB', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA A800-SXM4-80GB', 'memory_total': 85899345920}], 'memory': {'total': 503.3061103820801}}
2024-01-22 09:12:01,921 INFO    HandlerThread:3635583 [system_monitor.py:probe():211] Finished collecting system info
2024-01-22 09:12:01,921 INFO    HandlerThread:3635583 [system_monitor.py:probe():214] Publishing system info
2024-01-22 09:12:01,921 DEBUG   HandlerThread:3635583 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2024-01-22 09:12:01,922 DEBUG   HandlerThread:3635583 [system_info.py:_save_pip():67] Saving pip packages done
2024-01-22 09:12:01,923 INFO    HandlerThread:3635583 [system_monitor.py:probe():216] Finished publishing system info
2024-01-22 09:12:01,925 DEBUG   SenderThread:3635583 [sender.py:send():379] send: files
2024-01-22 09:12:01,925 INFO    SenderThread:3635583 [sender.py:_save_file():1375] saving file wandb-metadata.json with policy now
2024-01-22 09:12:01,934 DEBUG   SenderThread:3635583 [sender.py:send():379] send: telemetry
2024-01-22 09:12:01,935 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: stop_status
2024-01-22 09:12:01,935 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: stop_status
2024-01-22 09:12:02,046 INFO    Thread-12 :3635583 [dir_watcher.py:_on_file_created():271] file/dir created: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/wandb-metadata.json
2024-01-22 09:12:02,046 INFO    Thread-12 :3635583 [dir_watcher.py:_on_file_created():271] file/dir created: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/wandb-summary.json
2024-01-22 09:12:02,047 INFO    Thread-12 :3635583 [dir_watcher.py:_on_file_created():271] file/dir created: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/requirements.txt
2024-01-22 09:12:03,001 INFO    wandb-upload_0:3635583 [upload_job.py:push():131] Uploaded file /tmp/tmp8ypxnzxowandb/8gam5gyc-wandb-metadata.json
2024-01-22 09:12:03,044 INFO    Thread-12 :3635583 [dir_watcher.py:_on_file_created():271] file/dir created: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/output.log
2024-01-22 09:12:05,045 INFO    Thread-12 :3635583 [dir_watcher.py:_on_file_modified():288] file/dir modified: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/output.log
2024-01-22 09:12:05,744 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: status_report
2024-01-22 09:12:06,046 INFO    Thread-12 :3635583 [dir_watcher.py:_on_file_modified():288] file/dir modified: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/output.log
2024-01-22 09:12:10,745 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: status_report
2024-01-22 09:12:15,746 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: status_report
2024-01-22 09:12:16,977 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: stop_status
2024-01-22 09:12:16,978 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: stop_status
2024-01-22 09:12:18,050 INFO    Thread-12 :3635583 [dir_watcher.py:_on_file_modified():288] file/dir modified: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/output.log
2024-01-22 09:12:21,222 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: status_report
2024-01-22 09:12:22,880 DEBUG   SenderThread:3635583 [sender.py:send():379] send: exit
2024-01-22 09:12:22,880 INFO    SenderThread:3635583 [sender.py:send_exit():584] handling exit code: 0
2024-01-22 09:12:22,880 INFO    SenderThread:3635583 [sender.py:send_exit():586] handling runtime: 21
2024-01-22 09:12:22,882 INFO    SenderThread:3635583 [sender.py:_save_file():1375] saving file wandb-summary.json with policy end
2024-01-22 09:12:22,883 INFO    SenderThread:3635583 [sender.py:send_exit():592] send defer
2024-01-22 09:12:22,883 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: defer
2024-01-22 09:12:22,883 INFO    HandlerThread:3635583 [handler.py:handle_request_defer():170] handle defer: 0
2024-01-22 09:12:22,884 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: defer
2024-01-22 09:12:22,884 INFO    SenderThread:3635583 [sender.py:send_request_defer():608] handle sender defer: 0
2024-01-22 09:12:22,884 INFO    SenderThread:3635583 [sender.py:transition_state():612] send defer: 1
2024-01-22 09:12:22,884 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: defer
2024-01-22 09:12:22,884 INFO    HandlerThread:3635583 [handler.py:handle_request_defer():170] handle defer: 1
2024-01-22 09:12:22,885 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: defer
2024-01-22 09:12:22,885 INFO    SenderThread:3635583 [sender.py:send_request_defer():608] handle sender defer: 1
2024-01-22 09:12:22,885 INFO    SenderThread:3635583 [sender.py:transition_state():612] send defer: 2
2024-01-22 09:12:22,885 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: defer
2024-01-22 09:12:22,885 INFO    HandlerThread:3635583 [handler.py:handle_request_defer():170] handle defer: 2
2024-01-22 09:12:22,885 INFO    HandlerThread:3635583 [system_monitor.py:finish():190] Stopping system monitor
2024-01-22 09:12:22,885 DEBUG   SystemMonitor:3635583 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2024-01-22 09:12:22,886 INFO    HandlerThread:3635583 [interfaces.py:finish():202] Joined cpu monitor
2024-01-22 09:12:22,886 DEBUG   SystemMonitor:3635583 [system_monitor.py:_start():166] Finished system metrics aggregation loop
2024-01-22 09:12:22,886 INFO    HandlerThread:3635583 [interfaces.py:finish():202] Joined disk monitor
2024-01-22 09:12:22,887 DEBUG   SystemMonitor:3635583 [system_monitor.py:_start():170] Publishing last batch of metrics
2024-01-22 09:12:22,910 INFO    HandlerThread:3635583 [interfaces.py:finish():202] Joined gpu monitor
2024-01-22 09:12:22,910 INFO    HandlerThread:3635583 [interfaces.py:finish():202] Joined memory monitor
2024-01-22 09:12:22,911 INFO    HandlerThread:3635583 [interfaces.py:finish():202] Joined network monitor
2024-01-22 09:12:22,911 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: defer
2024-01-22 09:12:22,911 INFO    SenderThread:3635583 [sender.py:send_request_defer():608] handle sender defer: 2
2024-01-22 09:12:22,911 INFO    SenderThread:3635583 [sender.py:transition_state():612] send defer: 3
2024-01-22 09:12:22,912 DEBUG   SenderThread:3635583 [sender.py:send():379] send: stats
2024-01-22 09:12:22,913 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: defer
2024-01-22 09:12:22,913 INFO    HandlerThread:3635583 [handler.py:handle_request_defer():170] handle defer: 3
2024-01-22 09:12:22,913 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: defer
2024-01-22 09:12:22,913 INFO    SenderThread:3635583 [sender.py:send_request_defer():608] handle sender defer: 3
2024-01-22 09:12:22,913 INFO    SenderThread:3635583 [sender.py:transition_state():612] send defer: 4
2024-01-22 09:12:22,913 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: defer
2024-01-22 09:12:22,913 INFO    HandlerThread:3635583 [handler.py:handle_request_defer():170] handle defer: 4
2024-01-22 09:12:22,914 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: defer
2024-01-22 09:12:22,914 INFO    SenderThread:3635583 [sender.py:send_request_defer():608] handle sender defer: 4
2024-01-22 09:12:22,914 INFO    SenderThread:3635583 [sender.py:transition_state():612] send defer: 5
2024-01-22 09:12:22,914 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: defer
2024-01-22 09:12:22,914 INFO    HandlerThread:3635583 [handler.py:handle_request_defer():170] handle defer: 5
2024-01-22 09:12:22,914 DEBUG   SenderThread:3635583 [sender.py:send():379] send: summary
2024-01-22 09:12:22,915 INFO    SenderThread:3635583 [sender.py:_save_file():1375] saving file wandb-summary.json with policy end
2024-01-22 09:12:22,915 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: defer
2024-01-22 09:12:22,915 INFO    SenderThread:3635583 [sender.py:send_request_defer():608] handle sender defer: 5
2024-01-22 09:12:22,915 INFO    SenderThread:3635583 [sender.py:transition_state():612] send defer: 6
2024-01-22 09:12:22,916 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: defer
2024-01-22 09:12:22,916 INFO    HandlerThread:3635583 [handler.py:handle_request_defer():170] handle defer: 6
2024-01-22 09:12:22,916 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: defer
2024-01-22 09:12:22,916 INFO    SenderThread:3635583 [sender.py:send_request_defer():608] handle sender defer: 6
2024-01-22 09:12:22,921 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: status_report
2024-01-22 09:12:23,051 INFO    Thread-12 :3635583 [dir_watcher.py:_on_file_modified():288] file/dir modified: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/wandb-summary.json
2024-01-22 09:12:23,192 INFO    SenderThread:3635583 [sender.py:transition_state():612] send defer: 7
2024-01-22 09:12:23,193 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: defer
2024-01-22 09:12:23,193 INFO    HandlerThread:3635583 [handler.py:handle_request_defer():170] handle defer: 7
2024-01-22 09:12:23,193 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: defer
2024-01-22 09:12:23,194 INFO    SenderThread:3635583 [sender.py:send_request_defer():608] handle sender defer: 7
2024-01-22 09:12:23,880 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: poll_exit
2024-01-22 09:12:24,052 INFO    Thread-12 :3635583 [dir_watcher.py:_on_file_modified():288] file/dir modified: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/config.yaml
2024-01-22 09:12:25,052 INFO    Thread-12 :3635583 [dir_watcher.py:_on_file_modified():288] file/dir modified: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/output.log
2024-01-22 09:12:25,722 INFO    SenderThread:3635583 [sender.py:transition_state():612] send defer: 8
2024-01-22 09:12:25,722 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: poll_exit
2024-01-22 09:12:25,723 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: defer
2024-01-22 09:12:25,723 INFO    HandlerThread:3635583 [handler.py:handle_request_defer():170] handle defer: 8
2024-01-22 09:12:25,723 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: defer
2024-01-22 09:12:25,723 INFO    SenderThread:3635583 [sender.py:send_request_defer():608] handle sender defer: 8
2024-01-22 09:12:25,723 INFO    SenderThread:3635583 [job_builder.py:build():281] Attempting to build job artifact
2024-01-22 09:12:25,724 INFO    SenderThread:3635583 [job_builder.py:_get_source_type():411] no source found
2024-01-22 09:12:25,724 INFO    SenderThread:3635583 [sender.py:transition_state():612] send defer: 9
2024-01-22 09:12:25,724 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: defer
2024-01-22 09:12:25,724 INFO    HandlerThread:3635583 [handler.py:handle_request_defer():170] handle defer: 9
2024-01-22 09:12:25,724 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: defer
2024-01-22 09:12:25,724 INFO    SenderThread:3635583 [sender.py:send_request_defer():608] handle sender defer: 9
2024-01-22 09:12:25,724 INFO    SenderThread:3635583 [dir_watcher.py:finish():358] shutting down directory watcher
2024-01-22 09:12:26,053 INFO    Thread-12 :3635583 [dir_watcher.py:_on_file_modified():288] file/dir modified: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/output.log
2024-01-22 09:12:26,053 INFO    SenderThread:3635583 [dir_watcher.py:finish():388] scan: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files
2024-01-22 09:12:26,054 INFO    SenderThread:3635583 [dir_watcher.py:finish():402] scan save: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/output.log output.log
2024-01-22 09:12:26,054 INFO    SenderThread:3635583 [dir_watcher.py:finish():402] scan save: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/wandb-summary.json wandb-summary.json
2024-01-22 09:12:26,054 INFO    SenderThread:3635583 [dir_watcher.py:finish():402] scan save: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/requirements.txt requirements.txt
2024-01-22 09:12:26,054 INFO    SenderThread:3635583 [dir_watcher.py:finish():402] scan save: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/config.yaml config.yaml
2024-01-22 09:12:26,054 INFO    SenderThread:3635583 [dir_watcher.py:finish():402] scan save: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/wandb-metadata.json wandb-metadata.json
2024-01-22 09:12:26,054 INFO    SenderThread:3635583 [sender.py:transition_state():612] send defer: 10
2024-01-22 09:12:26,056 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: defer
2024-01-22 09:12:26,059 INFO    HandlerThread:3635583 [handler.py:handle_request_defer():170] handle defer: 10
2024-01-22 09:12:26,065 DEBUG   SenderThread:3635583 [sender.py:send_request():406] send_request: defer
2024-01-22 09:12:26,065 INFO    SenderThread:3635583 [sender.py:send_request_defer():608] handle sender defer: 10
2024-01-22 09:12:26,065 INFO    SenderThread:3635583 [file_pusher.py:finish():159] shutting down file pusher
2024-01-22 09:12:26,453 ERROR   wandb-upload_1:3635583 [internal_api.py:upload_file():2496] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/zjylab/UPRTH/ke2nwbty/output.log?Expires=1706001146&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=JiLSzzLJqmWFvSfl6SVhgdmyBrqgAz93LMac%2FOMI7CM9wXZs2jEmR5iJ%2FwZ8WfDpTEMniwULtOQCbKQExw44b%2FMOjCu3MNHPCpeYUkc9k53Nolu%2FT1NbkaHjfB8CgFwtGyXs6NOU1Q1Bn3vlgKj0HMk8y8mgJJ2lLRZsCXssB5Ks%2FucOqHBPsxWH5UkUkEEO8h4iyOqtIvisbjCpxvCeuGg74ARJgr4ZkBUJIQyd9gwfjA%2Bgj9C%2F6oxLfP5Tj%2FGTPDrJDBfD4e%2BXQnWbO3U7bOzeJ0NUFV0%2FJpC7HVwbW0CHo3ORE8B3H65xd359JNvpZhgYgeVa7PnKCz5ppTGaTA%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-22 09:12:26,454 ERROR   wandb-upload_1:3635583 [internal_api.py:upload_file():2498] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '478'}
2024-01-22 09:12:26,454 ERROR   wandb-upload_1:3635583 [internal_api.py:upload_file():2500] upload_file response body: 
2024-01-22 09:12:26,600 INFO    wandb-upload_2:3635583 [upload_job.py:push():131] Uploaded file /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/requirements.txt
2024-01-22 09:12:26,865 ERROR   wandb-upload_3:3635583 [internal_api.py:upload_file():2496] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/zjylab/UPRTH/ke2nwbty/config.yaml?Expires=1706001146&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=i%2FfsNpJHeLuEiZa%2BWRtJijW6t5HaO12eJG1ZMeSR%2F9ihhxfFtdymItRPvoN6%2Bcvq7zvOj%2BmNDYEtyPVrBx1arB9A%2FDhy0e5zCiJH3B0DCl1y%2BmRZ7NNRje7OPX3H3FtqrIJwKZxmmOyDJhI%2BERsk0lKu%2FzkVLjEi0HTwYYlr0BrWXAl0ONueEIYcQ4xoX7e8%2FWEK36Mokdco1yu19drNO%2BV6WxFpQMvUDZHxFsGVlsg6M7EiEhprm5rv80yo7E129V0ZzzDhm2I1qNvRDZ0hoPJrHwHCfxTxQ7%2Bbngyzt5jg9O0POQO8Wz%2B6J3Kb3GoB9zY5AJBpJq8P0ffnHoi4bg%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-22 09:12:26,866 ERROR   wandb-upload_3:3635583 [internal_api.py:upload_file():2498] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '693'}
2024-01-22 09:12:26,866 ERROR   wandb-upload_3:3635583 [internal_api.py:upload_file():2500] upload_file response body: 
2024-01-22 09:12:27,040 INFO    wandb-upload_0:3635583 [upload_job.py:push():131] Uploaded file /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/wandb-summary.json
2024-01-22 09:12:27,970 INFO    wandb-upload_1:3635583 [upload_job.py:push():131] Uploaded file /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/output.log
2024-01-22 09:12:28,061 ERROR   wandb-upload_3:3635583 [internal_api.py:upload_file():2496] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/zjylab/UPRTH/ke2nwbty/config.yaml?Expires=1706001146&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=i%2FfsNpJHeLuEiZa%2BWRtJijW6t5HaO12eJG1ZMeSR%2F9ihhxfFtdymItRPvoN6%2Bcvq7zvOj%2BmNDYEtyPVrBx1arB9A%2FDhy0e5zCiJH3B0DCl1y%2BmRZ7NNRje7OPX3H3FtqrIJwKZxmmOyDJhI%2BERsk0lKu%2FzkVLjEi0HTwYYlr0BrWXAl0ONueEIYcQ4xoX7e8%2FWEK36Mokdco1yu19drNO%2BV6WxFpQMvUDZHxFsGVlsg6M7EiEhprm5rv80yo7E129V0ZzzDhm2I1qNvRDZ0hoPJrHwHCfxTxQ7%2Bbngyzt5jg9O0POQO8Wz%2B6J3Kb3GoB9zY5AJBpJq8P0ffnHoi4bg%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-22 09:12:28,061 ERROR   wandb-upload_3:3635583 [internal_api.py:upload_file():2498] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '693'}
2024-01-22 09:12:28,061 ERROR   wandb-upload_3:3635583 [internal_api.py:upload_file():2500] upload_file response body: 
2024-01-22 09:12:28,061 INFO    wandb-upload_3:3635583 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "/usr/local/lib/python3.10/dist-packages/urllib3/connection.py", line 414, in connect
    self.sock = ssl_wrap_socket(
  File "/usr/local/lib/python3.10/dist-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/usr/local/lib/python3.10/dist-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/usr/lib/python3.10/ssl.py", line 513, in wrap_socket
    return self.sslsocket_class._create(
  File "/usr/lib/python3.10/ssl.py", line 1071, in _create
    self.do_handshake()
  File "/usr/lib/python3.10/ssl.py", line 1342, in do_handshake
    self._sslobj.do_handshake()
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/requests/adapters.py", line 489, in send
    resp = conn.urlopen(
  File "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py", line 787, in urlopen
    retries = retries.increment(
  File "/usr/local/lib/python3.10/dist-packages/urllib3/util/retry.py", line 550, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/usr/local/lib/python3.10/dist-packages/urllib3/packages/six.py", line 769, in reraise
    raise value.with_traceback(tb)
  File "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py", line 1042, in _validate_conn
    conn.connect()
  File "/usr/local/lib/python3.10/dist-packages/urllib3/connection.py", line 414, in connect
    self.sock = ssl_wrap_socket(
  File "/usr/local/lib/python3.10/dist-packages/urllib3/util/ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/usr/local/lib/python3.10/dist-packages/urllib3/util/ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/usr/lib/python3.10/ssl.py", line 513, in wrap_socket
    return self.sslsocket_class._create(
  File "/usr/lib/python3.10/ssl.py", line 1071, in _create
    self.do_handshake()
  File "/usr/lib/python3.10/ssl.py", line 1342, in do_handshake
    self._sslobj.do_handshake()
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/internal/internal_api.py", line 2491, in upload_file
    response = self._upload_file_session.put(
  File "/usr/local/lib/python3.10/dist-packages/requests/sessions.py", line 647, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/requests/adapters.py", line 547, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/internal/internal_api.py", line 2520, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/usr/local/lib/python3.10/dist-packages/wandb/sdk/internal/internal_api.py", line 2491, in upload_file
    response = self._upload_file_session.put(
  File "/usr/local/lib/python3.10/dist-packages/requests/sessions.py", line 647, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/requests/adapters.py", line 547, in send
    raise ConnectionError(err, request=request)
wandb.sdk.lib.retry.TransientError: None
2024-01-22 09:12:28,882 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: keepalive
2024-01-22 09:12:30,362 ERROR   wandb-upload_3:3635583 [internal_api.py:upload_file():2496] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/zjylab/UPRTH/ke2nwbty/config.yaml?Expires=1706001146&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=i%2FfsNpJHeLuEiZa%2BWRtJijW6t5HaO12eJG1ZMeSR%2F9ihhxfFtdymItRPvoN6%2Bcvq7zvOj%2BmNDYEtyPVrBx1arB9A%2FDhy0e5zCiJH3B0DCl1y%2BmRZ7NNRje7OPX3H3FtqrIJwKZxmmOyDJhI%2BERsk0lKu%2FzkVLjEi0HTwYYlr0BrWXAl0ONueEIYcQ4xoX7e8%2FWEK36Mokdco1yu19drNO%2BV6WxFpQMvUDZHxFsGVlsg6M7EiEhprm5rv80yo7E129V0ZzzDhm2I1qNvRDZ0hoPJrHwHCfxTxQ7%2Bbngyzt5jg9O0POQO8Wz%2B6J3Kb3GoB9zY5AJBpJq8P0ffnHoi4bg%3D%3D: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))
2024-01-22 09:12:30,362 ERROR   wandb-upload_3:3635583 [internal_api.py:upload_file():2498] upload_file request headers: {'User-Agent': 'python-requests/2.28.1', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '693'}
2024-01-22 09:12:30,362 ERROR   wandb-upload_3:3635583 [internal_api.py:upload_file():2500] upload_file response body: 
2024-01-22 09:12:32,066 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: status_report
2024-01-22 09:12:33,106 WARNING StreamThr :3635583 [internal.py:is_dead():414] Internal process exiting, parent pid 3635280 disappeared
2024-01-22 09:12:33,106 ERROR   StreamThr :3635583 [internal.py:wandb_internal():152] Internal process shutdown.
2024-01-22 09:12:33,884 DEBUG   HandlerThread:3635583 [handler.py:handle_request():144] handle_request: keepalive
2024-01-22 09:12:33,884 INFO    HandlerThread:3635583 [handler.py:finish():861] shutting down handler
2024-01-22 09:12:34,067 INFO    SenderThread:3635583 [sender.py:finish():1552] shutting down sender
2024-01-22 09:12:34,067 INFO    SenderThread:3635583 [file_pusher.py:finish():159] shutting down file pusher
2024-01-22 09:12:34,067 INFO    SenderThread:3635583 [file_pusher.py:join():164] waiting for file pusher
2024-01-22 09:12:34,067 INFO    WriterThread:3635583 [datastore.py:close():294] close: /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/run-ke2nwbty.wandb
2024-01-22 09:12:34,948 INFO    wandb-upload_3:3635583 [upload_job.py:push():131] Uploaded file /workspace/UPRTH/wandb/run-20240122_091200-ke2nwbty/files/config.yaml
2024-01-22 09:12:35,149 INFO    Thread-11 (_thread_body):3635583 [sender.py:transition_state():612] send defer: 11
2024-01-22 09:12:35,569 INFO    SenderThread:3635583 [file_stream.py:finish():593] file stream finish called
2024-01-22 09:12:35,809 INFO    SenderThread:3635583 [file_stream.py:finish():597] file stream finish is done
